.github/workflows/module_integration.yml

```diff
@@ -133,3 +133,73 @@ jobs:
           . .venv/bin/activate
           ./cmake_build/test/test_routing_pybind
           deactivate
+  
+  test_forcing_engine_integration:
+    # TODO: Remove when integration is ready
+    if: false
+
+    name: Test Forcings Engine Integration
+    runs-on: 'ubuntu-latest'
+    env:
+      CC: mpicc
+      CXX: mpic++
+      FC: gfortran
+      F90: gfortran
+
+    steps:
+      - name: Checkout NGen
+        uses: actions/checkout@v4
+      
+      - name: Checkout Forcings Engine
+        uses: actions/checkout@v4
+        with:
+          # TODO: replace once merged, https://github.com/NOAA-OWP/ngen-forcing/pull/9
+          # repository: NOAA-OWP/ngen-forcing
+          repository: jduckerOWP/ngen-forcing
+          ref: master
+          path: extern/ngen-forcing
+          clean: false
+
+      - name: Set up Spack
+        uses: spack/setup-spack@v2
+        with:
+          buildcache: true
+
+      - name: Install Forcings Engine Dependencies
+        run: |
+          spack install parallel-netcdf
+          spack install netcdf-c +parallel-netcdf
+          spack install netcdf-fortran
+          spack install parallelio +pnetcdf +ncint
+          spack install esmf@8.5.0 +pnetcdf
+      
+      - name: Install ESMPy
+        run: |
+          git clone -b v8.5.0 --depth 1 https://github.com/esmf-org/esmf /tmp/esmf
+          pushd /tmp/esmf/src/addon/esmpy
+          spack load esmf
+          python3 -m pip install .
+          popd
+          rm -r /tmp/esmf
+      
+      # TODO: Store in Git or Pull? They are large files
```

Comment:
Git-LFS could be good for this.
https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-git-large-file-storage

Reply:
Or, we could do the testing with very cut-down files that will fit in the repo more nicely.

Reply:
Yeah those are the same two options I was considering -- I'm leaning toward creating cut down files and creating a simple domain